roles/matrix-nginx-proxy/templates/nginx/conf.d/matrix-synapse.conf.j2

```diff
@@ -12,7 +19,7 @@
 		{% if use_worker_affinity %}
 			# ensures that requests from the same client will always be passed
 			# to the same server (except when this server is unavailable)
-			hash $http_x_forwarded_for;
+			hash $http_authorization;
```

Comment:
Could this conflict with the idea of workers per sub type?

Reply:
It could, only if we mix some generic workers with `complete` and other sub-types. (not considered in default scenario).